elementary/monitor/api/tests/tests.py

```diff
@@ -267,20 +267,28 @@ def _get_invocations(
         self, test_result_db_rows: List[TestResultDBRowSchema]
     ) -> Dict[TestUniqueIdType, InvocationsSchema]:
         grouped_invocations = defaultdict(list)
+        grouped_invocation_ids = defaultdict(list)
         for test_result_db_row in test_result_db_rows:
             try:
                 elementary_unique_id = test_result_db_row.elementary_unique_id
-                grouped_invocations[elementary_unique_id].append(
-                    InvocationSchema(
-                        id=test_result_db_row.invocation_id
-                        or test_result_db_row.test_execution_id,
-                        time_utc=test_result_db_row.detected_at,
-                        status=test_result_db_row.status,
-                        affected_rows=self._parse_affected_row(
-                            results_description=test_result_db_row.test_results_description
-                        ),
-                    )
+                invocation_id = (
+                    test_result_db_row.invocation_id
```

Comment:
Why is there an `or` expression here?
I think there will always be an `invocation_id` and a `test_execution_id` in `elementary_test_results`.

Reply:
It's a backwards compatibility 🙂 
On older versions of our dbt package we didn't have invocation id at the `elementary_test_results` table, and because that is an incremental table, we have cases where invocation id is `None`.
I do prefer moving into invocation id and not test execution id, so I left it as a fall back (This is actually a part of an older PR).

Reply:
Why not use `test_execution_id` for all?